src/stores/widgets/StopGapWidget.ts

```diff
@@ -151,6 +158,7 @@ export class StopGapWidget extends EventEmitter {
     private mockWidget: ElementWidget;
     private scalarToken?: string;
     private roomId?: string;
+    private viewedRoomId: string | null = null;
```

Comment:
This could do with a comment I think, ie. it's the last room id we told the widget it was getting events for, and only used for account widgets.